sdk/cosmosdb/cosmos/samples-dev/ChangeFeedIterator/ChangeFeedFullFidelityEntireContainer.ts

```diff
@@ -0,0 +1,106 @@
+
+logSampleHeader("Change Feed For Entire Container With AllVersionsAndDeletes Mode");
+
+async function ingestData(container: Container, initialize: number, end: number) {
+  console.log(`beginning data ingestion from id - item${initialize} to id - item${end - 1}`);
+  for (let i = initialize; i < end; i++) {
+    await container.items.create({ id: `item${i}`, name: `sample1`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample2`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample3`, key: i });
+    await container.items.create({ id: `item${i}`, name: `sample4`, key: i });
+  }
+  console.log("ingested items");
+}
+
+async function insertAndModifyData(container: Container, initialize: number, end: number) {
+  await ingestData(container, initialize, end);
+  await container.items.upsert({ id: `item${initialize}`, name: `sample1`, key: initialize + 1 });
+  console.log(`upserted item with id - item${initialize} and partition key - sample1`);
+  await container.item(`item${initialize}`, `sample1`).delete();
+  console.log(`deleted item with id - item${initialize} and partition key - sample1`);
+}
+
+// Establish a new instance of the CosmosClient to be used throughout this demo
+const client = new CosmosClient({ endpoint, key });
+
+async function run(): Promise<void> {
+  try {
+    const { database } = await client.databases.createIfNotExists({ id: databaseId });
+    const containerDef = {
+      id: containerId,
+      partitionKey: {
+        paths: ["/name"],
+        version: PartitionKeyDefinitionVersion.V1,
+      },
+      throughput: 11000,
+    };
+    const { container } = await database.containers.createIfNotExists(containerDef);
+    let changeFeedIteratorOptions: ChangeFeedIteratorOptions;
+    changeFeedIteratorOptions = {
+      changeFeedStartFrom: ChangeFeedStartFrom.Now(),
+      changeFeedMode: ChangeFeedMode.AllVersionsAndDeletes,
+    };
+
+    let iterator = container.items.getChangeFeedIterator(changeFeedIteratorOptions);
+    logStep("Start fetching changes from now");
+    let continuationToken = "";
+    let res = await iterator.readNext();
+    console.log("Result should be empty as no changes are made yet", res.result);
+    await insertAndModifyData(container, 1, 3);
+    while (iterator.hasMoreResults) {
+      const res = await iterator.readNext();
+      if (res.statusCode === StatusCodes.NotModified) {
+        continuationToken = res.continuationToken;
+        break;
+      }
+      console.log("Results Found: ", res.result);
+    }
+
+    await insertAndModifyData(container, 3, 5);
+    changeFeedIteratorOptions = {
+      maxItemCount: 5,
+      changeFeedStartFrom: ChangeFeedStartFrom.Continuation(continuationToken),
+      changeFeedMode: ChangeFeedMode.AllVersionsAndDeletes,
+    };
+    iterator = container.items.getChangeFeedIterator(changeFeedIteratorOptions);
+    console.log("Start fetching changes from continuation token");
```

Comment:
What’s the purpose of adding a sample to start fetching changes from a continuation token? Are we recommending this approach? Also, I think all these change feed samples could be combined into the same file with different log steps.

Reply:
The AllVersionsAndDeletes mode is supported for ChangeFeedStartFrom.Now and ChangeFeedStatrtFrom.Continuation. So, sample for both is added. 
Basically, this file contains sample for ChangeFeedIterator for entire container in all versions and delete mode. Same for other two iterators.